src/intl/ru-RU.JS

```diff
@@ -193,7 +193,7 @@ export default {
   pinPage: 'Закрепить {label}',
   // Status composition
   composeStatus: 'Создать запись',
-  postStatus: 'Опубликовать!',
+  postStatus2: 'Опубликовать!',
```

Comment:
so this will break ru locale?

Reply:
Looks like a mistake yes, thank you @Oreolek